Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ruby:2.3.7
+RUN apt-get -q update && apt-get -y install git make gcc
+WORKDIR /usr/src/app
+COPY Gemfile Gemfile.lock ./
+RUN bundle install
+COPY . .
+RUN rake db:schema:load && rake db:migrate
```

Comment:
You don't have to run `rake db:migrate`. Let's say "it's already included in first part".